src/lib/math/pcurves/pcurves_impl/pcurves_wrap.h

```diff
@@ -52,20 +44,33 @@ class PrimeOrderCurveImpl final : public PrimeOrderCurve {
       }
 
       ProjectivePoint mul(const AffinePoint& pt, const Scalar& scalar, RandomNumberGenerator& rng) const override {
-         auto tbl = WindowedMulTable<C, 4>(from_stash(pt));
+         auto tbl = WindowedMulTable<C, VarPointWindowBits>(from_stash(pt));
          return stash(tbl.mul(from_stash(scalar), rng));
       }
 
       secure_vector<uint8_t> mul_x_only(const AffinePoint& pt,
                                         const Scalar& scalar,
                                         RandomNumberGenerator& rng) const override {
-         auto tbl = WindowedMulTable<C, 4>(from_stash(pt));
+         auto tbl = WindowedMulTable<C, VarPointWindowBits>(from_stash(pt));
          auto pt_x = to_affine_x<C>(tbl.mul(from_stash(scalar), rng));
          secure_vector<uint8_t> x_bytes(C::FieldElement::BYTES);
          pt_x.serialize_to(std::span<uint8_t, C::FieldElement::BYTES>{x_bytes});
          return x_bytes;
       }
 
+      class PrecomputedMul2TableC final : public PrimeOrderCurve::PrecomputedMul2Table {
+         public:
+            static constexpr size_t WindowBits = 3;
```

Comment:
Did you mean to also replace this not-quite-so magic number as well? To me it now seems orphaned.